include/psa/crypto_compat.h

```diff
@@ -0,0 +1,102 @@
+/**
+ * \file psa/crypto_compat.h
+ *
+ * \brief PSA cryptography module: Backward compatibility aliases
```

Comment:
I would add a sentence about this file also having deprecation information.

Reply:
What do you mean? Something like “This header declares alternative names for macros and functions that may be removed in a future version of Mbed Crypto”?

Reply:
Or just `Backward compatibility aliases and deprecated entities`